app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java

```diff
@@ -61,7 +60,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         SharedPreferences.Editor editor1 = num1.edit();
 
         SharedPreferences num1g = this.getActivity().getSharedPreferences("phonenum1", Context.MODE_PRIVATE);
-        String value1 = num1g.getString("num1","Enter num1");
+        String value1 = num1g.getString("num1", "Enter Num1");
```

Comment:
rename these value1, value2, value3 too.. into meaningful names for better understanding

Reply:
okay